docs/articles/infrastructure/ci-cd/freeipa.rst

```diff
@@ -316,4 +484,87 @@ Try to identify an LDAP user:
     # Should return "no such user"
     id someldapuser
 
+Replica
+^^^^^^^
```

Comment:
I would prefer some headers within this section. If possible, it makes it easier to scan through faster without having to text-search.

Reply:
Added that but it starts to feel like an american research paper with infinite levels of subsections.

Reply:
Haha sorry. I don't feel it's necessary, but I prefer it since it usually makes the process of troubleshooting a little easier.